### PR TITLE
Add unit test for VolumeList

### PR DESCRIPTION
**Pull request by vdemeester**
And time for `VolumeList` 🐧.

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by calavera** [comment]
:panda_face: LGTM

File: client/volume_list_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	}
+}
+
+func TestVolumeList(t *testing.T) {
+	expectedURL := "/volumes"
+
+	filters := filters.NewArgs()
+	filters.Add("label", "label1")
+	filters.Add("label", "label2")
+	filters.Add("dangling", "true")
```
Comment (duglin) [inline client/volume_list_test.go]: did you want to test the dangling=false side of this too? like that other PR

Comment (vdemeester) [inline client/volume_list_test.go]: damn.. did it again :sweat_smile:… updating :wink:

**Comment by vdemeester** [comment]
@duglin updated ;)

**Comment by duglin** [comment]
thanks!
LGTM

**Pull request merged**